actioncable/blade.yml

```diff
@@ -5,3 +5,9 @@ load_paths:
 
 logical_paths:
   - test.js
+
+build:
+  logical_paths:
+    - action_cable.js
+  path: lib/assets/compiled
+  js_compressor: uglifier
```

Comment:
I removed js compression for now (https://github.com/javan/rails/commit/27ae7c326556b206aee4b929fe1f2061553d85a4), but I think we have it backwards. The reasons listed for using an uncompressed version are all development concerns so, if anything, we should provide an additional uncompressed version for debugging.

Turbolinks is part of the default stack and [compresses its source](https://github.com/turbolinks/turbolinks-source-gem/blob/master/lib/assets/javascripts/turbolinks.js) so we're currently inconsistent about the js we bundle.

<blockquote>it doesn't really matter -- it'll be compressed eventually either way</blockquote>

<blockquote>Anyone doing doing this will already be using compression by some other means</blockquote>

You both seem pretty confident about this for some reason. Have you seen the... internet? 😁